extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -146,9 +146,9 @@ export const done = localize('done', "Done");
 export const nameMustNotBeEmpty = localize('nameMustNotBeEmpty', "Name must not be empty");
 
 // Publish Dialog options
-export const publishOptions = localize('publishOptions', 'Publish Options');
-export const publishingOptions = localize('publishingOptions', 'Publishing Options');
-export const GeneralOptions: string = localize('generalOptions', "General Options");
+export const AdvancedOptionsButton = localize('advancedOptionsButton', 'Advanced...');
+export const AdvancedPublishOptions = localize('advancedPublishOptions', 'Advanced Publish Options');
+export const PublishingOptions = localize('publishingOptions', 'Publishing Options');
```

Comment:
where is this string used? I don't see it in the screenshots. As discussed in the UX meeting, let's use "Publish" instead of "Publishing"

Reply:
Oh I have those changes along with the include objects PR, I guess, since its a single tab and is not showing the heading "Publish Options" but when i open the Include objects code its is showing the both tabs with :).